src/Foundation/RequestInputHandler.php

```diff
@@ -2,12 +2,24 @@
 
 namespace WeStacks\TeleBot\Foundation;
 
+use WeStacks\TeleBot\Objects\Chat;
 use WeStacks\TeleBot\Objects\User;
 use WeStacks\TeleBot\TeleBot;
 
 abstract class RequestInputHandler extends UpdateHandler
 {
-    private static function storage(TeleBot $bot): StorageContract
```

Comment:
What is the case, where you'd want this to be protected instead of private? I believe storage should be centralized for the whole bot and there are no cases to override it.